pkg/controller/template/test_data/templates/master/01-master-kubelet/aws/units/kubelet.service

```diff
@@ -25,7 +25,7 @@ contents: |
         --client-ca-file=/etc/kubernetes/ca.crt \
         --cloud-provider=aws \
         --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec \
-        \
+         \
```

Comment:
?

Reply:
because of change [here](https://github.com/openshift/machine-config-operator/pull/591/files#diff-d5047a0acdc71dcfb6fc0b86a556fd4eR29)